yarn.lock

```diff
@@ -6,12 +6,12 @@ __metadata:
   cacheKey: 8
```

Comment:
AFAICT, there should not have been changes here checked in, since only the README.md was updated. Should be reverted.